index.js

```diff
@@ -29,7 +29,9 @@ PostgresInterval.prototype.toPostgres = function () {
         value = (value + this.milliseconds / 1000).toFixed(6).replace(/\.?0+$/, '')
       }
 
-      return value + ' ' + property
+      const isSingular = value === 1 || value === '1'
```

Comment:
Where is `value` becoming a string? Thought it might be L29 since that looks like `Number + Number / String` but that evaluates to a number in my simple testing:


```sh
node -p "typeof(1 + 1 / '2')"
# => 'number'
```

Reply:
The `toFixed` returns a `string`, I thought that we could wrap that instead with a `parseFloat`, but I did not see a problem doing it this way (probably one extra check is better for performance than parsing).

Reply:
Ok yeah just was mis-reading, saw `isFixed` as part of a divisor

Reply:
```suggestion
      // fractional seconds will be a String, all others are Number
      const isSingular = String(value) === '1'
```